Library/Homebrew/cask/cmd.rb

```diff
@@ -221,9 +221,14 @@ def run(*_args)
         usage
 
         return if @command.nil?
-        return if @command == "help" && @args.empty?
 
-        raise ArgumentError, "help does not take arguments."
+        if @command == "help"
+          return if @args.empty?
+
+          raise ArgumentError, "help does not take arguments." if @args.length
+        end
+
+        raise ArgumentError, "brew cask does not recognise this command"
```

Comment:
For consistent with `brew`'s invalid command error messages, I think this should be:
```
raise ArgumentError, "Unknown Cask command: #{command}"
```

If you want to give me write access to your repo I'll happily commit that.

Reply:
Will do. Go ahead and make the change. Thanks

Reply:
Have added you as collab -> https://github.com/jpninanjohn/brew/invitations